function/index.js

```diff
@@ -15,22 +14,37 @@ function serialize(object) {
 function parseLocation(response) {
     const body = JSON.parse(response.Messages[0].Body).Records[0];
 
-
     const bucket = body.s3.bucket.name;
     const key = metadataService.unescape(decodeURIComponent(body.s3.object.key));
 
-    let arr = key.split("\\");
+    const arr = key.split('\\');
     arr.pop();
-    const dir = arr.join("\\");
+    const dir = arr.join('\\');
 
     return {
         Bucket: bucket,
         Directory: dir
     };
 }
 
-async function handler(event, context) {
+function validateFiles(messageObjects) {
+    // check length of message queue is only 2
+    if (messageObjects.length > 2) {
+        throw Error('More than two files - there should be two');
```

Comment:
Can we log the specific length of the messageObjects, it will help with debugging later down the line.